StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/UpdatePaymentMethodViewController.swift

```diff
@@ -0,0 +1,378 @@
+        }
+        return nil
+    }()
+
+    private lazy var cardSection: SectionElement = {
+        lazy var panElement: TextFieldElement = {
+            return TextFieldElement.LastFourConfiguration(lastFour: viewModel.paymentMethod.card?.last4 ?? "", cardBrandDropDown: cardBrandDropDown).makeElement(theme: appearance.asElementsTheme)
+        }()
+        lazy var expiryDateElement: TextFieldElement = {
+            let expiryDate = CardExpiryDate(month: viewModel.paymentMethod.card?.expMonth ?? 0, year: viewModel.paymentMethod.card?.expYear ?? 0)
+            let expiryDateElement = TextFieldElement.ExpiryDateConfiguration(defaultValue: expiryDate.displayString, isEditable: false).makeElement(theme: appearance.asElementsTheme)
+            return expiryDateElement
+
+        }()
+        lazy var cvcElement: TextFieldElement = {
+            let cvcConfiguration = TextFieldElement.CensoredCVCConfiguration(brand: self.viewModel.paymentMethod.card?.preferredDisplayBrand ?? .unknown)
+            let cvcElement = cvcConfiguration.makeElement(theme: appearance.asElementsTheme)
+            return cvcElement
+
+        }()
+        let allSubElements: [Element?] = [
+            panElement,
+            SectionElement.HiddenElement(cardBrandDropDown),
+            SectionElement.MultiElementRow([expiryDateElement, cvcElement])
+        ]
+        let section = SectionElement(elements: allSubElements.compactMap { $0 }, theme: appearance.asElementsTheme)
+        section.delegate = self
+        return section
+    }()
+
+    private lazy var usBankAccountSection: UIStackView = {
+        lazy var nameElement: SectionElement = {
+            return SectionElement(elements: [TextFieldElement.NameConfiguration(defaultValue: viewModel.paymentMethod.billingDetails?.name, isEditable: false).makeElement(theme: appearance.asElementsTheme)])
+        }()
+        lazy var emailElement: SectionElement = {
+            return SectionElement(elements: [TextFieldElement.EmailConfiguration(defaultValue: viewModel.paymentMethod.billingDetails?.email, isEditable: false).makeElement(theme: appearance.asElementsTheme)])
+        }()
+        lazy var bankAccountElement: SectionElement = {
+            return SectionElement(elements: [TextFieldElement.USBankNumberConfiguration(bankName: viewModel.paymentMethod.usBankAccount?.bankName ?? "Bank name", lastFour: viewModel.paymentMethod.usBankAccount?.last4 ?? "").makeElement(theme: appearance.asElementsTheme)])
+        }()
+        let stackView = UIStackView(arrangedSubviews: [nameElement.view, emailElement.view, bankAccountElement.view])
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.axis = .vertical
+        stackView.setCustomSpacing(8, after: nameElement.view) // custom spacing from figma
+        stackView.setCustomSpacing(8, after: emailElement.view) // custom spacing from figma
+        return stackView
+    }()
+
+    private lazy var sepaDebitSection: UIStackView = {
```

Comment:
I think it would be good to move all the form creation for card, us bank account and sepa into it's own class.

Then it can have a function where you pass it in a view model and it just returns a view.